content/docs/getting_started/install.md

```diff
@@ -1,16 +1,17 @@
 ---
-title: "Installation"
-description: "This section describes how to install/uninstall Open Service Mesh (OSM) on a Kubernetes cluster"
+title: 'Installation'
+description: 'This section describes how to install/uninstall Open Service Mesh (OSM) on a Kubernetes cluster'
```

Comment:
What's the purpose of changing this to single quotes? If there's no difference, please revert so that it's consistent with other docs.